3rdPartyCypress/cypress/integration/worldServiceCoUkRedirect.js

```diff
@@ -0,0 +1,36 @@
+import services from '../../../src/app/lib/config/services';
+
+Object.keys(services).forEach(service => {
+  const notWSServices = ['news', 'cymrufyw', 'naidheachdan', 'default']; // Not WS
+
+  if (notWSServices.includes(service)) {
+    return;
+  }
+  describe('WS Redirects', () => {
+    it(`should redirect *bbc.com/${service}`, () => {
+      const urlsTotest = [
+        `https://www.bbc.co.uk/${service}`,
```

Comment:
We'll be testing these live urls on each environment `local`, `test`, `live`. We should wrap in a conditional `if (Cypress.env('APP_ENV') !== 'local') {}` block, then update these paths to be `https://www.test.bbc.co.uk` on the test env and `https://www.bbc.co.uk` on the live env. Best place for this is to add field `ukBasePath` to `/support/config/envs.js`

Reply:
Have updated to support this @sareh

Reply:
@sareh this has been removed now as we do not need to run this tests on different env.

Reply:
Okay, thanks, Chris.